x/crosschain/keeper/msg_server_migrate_tss_funds_test.go

```diff
@@ -35,20 +35,38 @@ func TestMsgServer_MigrateTssFunds(t *testing.T) {
 		_, found := k.GetCrossChainTx(ctx, index)
```

Comment:
We should consider checking in the test that amount is decreased by the fee.

Reply:
1a24f7336d587480ed91c3bb65541a2fdecc01ae